var/spack/repos/builtin/packages/sst-core/package.py

```diff
@@ -73,6 +74,12 @@ class SstCore(AutotoolsPackage):
     depends_on("m4", type="build", when="@master:")
     depends_on("gettext")
 
+    sanity_check_is_file = [
+        join_path("bin", "sst-config"),
+        join_path("bin", "sst-test-core"),
+        join_path("bin", "sst-test-elements"),
+    ]
```

Comment:
Could you please add a comment as to why this is necessary? I am assuming this is due to some shortcoming in SST's autotools set-up?

Reply:
It's not really necessary so I've removed it.